tests/cmdline/repl_autocomplete_underscore.py

```diff
@@ -0,0 +1,34 @@
+# Test REPL autocompletion filtering of underscore attributes
+# This test is designed to test behavior with frozen code when run with --via-mpy
+
+# Start paste mode
+\x05
+class TestClass:
+    def __init__(self):
+        self.public_attr = 1
+        self._private_attr = 2
+        self.__very_private = 3
+    
+    def public_method(self):
+        pass
+    
+    def _private_method(self):
+        pass
+    
+    @property
+    def public_property(self):
+        return 42
+    
+    @property  
+    def _private_property(self):
+        return 99
+
+\x04
+# Paste executed
+
+# Create an instance
+obj = TestClass()
+
+# Test tab completion on the instance
+# The tab character after `obj.` below triggers the completion
+obj.	a	
```

Comment:
Could it use the escape sequence for tab here, so it's clearer?  (Assuming the escape sequence is needed for paste mode, may as well use it here as well.)